### PR TITLE
Use token for version

### PR DESCRIPTION
**Pull request by codyhoag**
Suggested in #19992

### CODE REVIEW

File: installing/install_config/configuring-private-cluster.adoc [hunk 1]
```diff
@@ -5,7 +5,7 @@ include::modules/common-attributes.adoc[]
 
 toc::[]
 
-After you install {product-title} version 4.2 cluster, you can set some of its core components to be private.
+After you install {product-title} version {product-version} cluster, you can set some of its core components to be private.
```
Comment (bergerhoffer) [inline installing/install_config/configuring-private-cluster.adoc]: The replace is fine. And I know it was like this before that, but it reads weird if you want to fix it while you're in there.

I'd either put "an" in front of `{product-title} `, or remove "cluster". So it either says:

```
After you install an OpenShift Container Platform version 4.4 cluster,
or
After you install OpenShift Container Platform version 4.4,
```

**Review by bergerhoffer — COMMENTED** [review]
Your update LGTM. But added a suggestion for an improvement on the existing text.

File: installing/install_config/configuring-private-cluster.adoc (hunk 1 above)
Comment (codyhoag) [reply installing/install_config/configuring-private-cluster.adoc]: Yep! Better to improve it while I'm in there. I'll fix the flow. Thanks!

Comment (bergerhoffer) [reply installing/install_config/configuring-private-cluster.adoc]: Thanks @codyhoag!

**Pull request merged**